src/aotcompile.cpp

```diff
@@ -74,15 +74,13 @@ STATISTIC(CreateNativeGlobals, "Number of globals compiled for jl_create_native"
 template<class T> // for GlobalObject's
 static T *addComdat(T *G)
 {
-#if defined(_OS_WINDOWS_)
```

Comment:
shouldn't this still need a guard? I don't know what `DLLExportStorageClass` will do if used on non-PE platforms